src/main/java/org/folio/oaipmh/helpers/AbstractHelper.java

```diff
@@ -66,6 +75,18 @@
    */
   protected StorageHelper storageHelper = StorageHelper.getInstance();
 
+  /**
+   * Creates basic {@link OAIPMH} with ResponseDate and Request details
+   * @param request {@link Request}
+   * @return basic {@link OAIPMH}
+   */
+  protected OAIPMH buildBaseResponse(Request request) {
```

Comment:
Please remove this. This logic was extracted to ResponseHelper::buildBaseOaipmhResponse. Any response building logic should bases on ResponseHelper class methods.